docs/notebooks/negative_binomial.ipynb

```diff
@@ -18,28 +18,36 @@
    "cell_type": "markdown",
```

Comment:
<p>I would replace "we can use the following definition:" with "the probability mass function (pmf) results:" and directly show the first pmf you list.</p><p> </p><p>Then I find it a bit repetitive that you say twice, and very close, that Y starts at zero when modeling failures, but it's fine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/884/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>